Source/WTF/wtf/text/StringView.h

```diff
@@ -197,6 +197,7 @@ class StringView final {
     explicit StringView(const char*);
 
     friend bool equal(StringView, StringView);
+    friend bool equal(StringView, StringView, unsigned);
```

Comment:
Please add the name `length` to the 3rd argument.  I think it adds meaningful info for someone who is only reading the header file.

Reply:
Added.